recipes/robocorp-excel/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "robocorp-excel" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/robocorp_excel-{{ version }}.tar.gz
+  sha256: c1cb8a3747dcd1d3cfee06d28455c02a9a93f71fe257026ac850c40450808a24
+
+build:
+  noarch: python
+  script: 
+    - echo f | xcopy /f /y /S %RECIPE_DIR%\README.md %SRC_DIR%\README.md  # [win]
+    - cp -R {{ RECIPE_DIR }}/README.md {{ SRC_DIR }}/README.md  # [unix]
```

Comment:
If this is really needed you can have just the "not win" one and let Windows fail b/c the recipe is noarch anyway and the only Windows build of it will be in staged-recipes.

However, my question is: Is this needed for the build? Kind of odd to required a file that is just info and it is not in the tarball.